zipline/data/loader.py

```diff
@@ -43,6 +43,8 @@
     (treasuries, 'treasury_curves.csv', 'www.federalreserve.gov'),
 }
 
+MARKET_DATA_DIR = 'zipline/resources/market_data'
```

Comment:
If you make these new directories packages, then `zipline.resources.market_data.__file__` would even work. Not sure if there were reasons we didn't do that already...